### ISSUE TITLE
LinkedCellEditor doesn't open

### ISSUE THREAD

**Issue by rjahn** [sibvisions/reactUI#484]
User: features
Screen: Components

Tried to open The Group editor and following exception occurs:

```
TypeError: Cannot read properties of undefined (reading 'map')
    at http://localhost:3000/static/js/bundle.js:16268:20
    at Function.getJSXElement (http://localhost:3000/static/js/bundle.js:238638:41)
    at createLabelItem (http://localhost:3000/static/js/bundle.js:198901:105)
    at createItem (http://localhost:3000/static/js/bundle.js:198956:18)
    at itemTemplate (http://localhost:3000/static/js/bundle.js:199013:26)
    at Function.getJSXElement (http://localhost:3000/static/js/bundle.js:238638:41)
    at createItem (http://localhost:3000/static/js/bundle.js:240548:77)
    at Array.map (<anonymous>)
    at createItems (http://localhost:3000/static/js/bundle.js:240555:18)
    at createContent (http://localhost:3000/static/js/bundle.js:240558:17)
```

**Comment by gimmixAT** [comment]
seems to work again in the Components screen. Are there any other screens I should check that contain it?

**Comment by rjahn** [comment]
Works with current master branch, but 2 different problems:

master_04_03_2024 doesn't show record on first click
-> open Components screen -> try to click the arrow of Group-1 cell editor -> nothing happens -> focus field -> press arrow -> 2 records shown

master:

opens on first click but header is shown and cell format is not applied

**Comment by gimmixAT** [comment]
regarding `master` - in the options `tableHeaderVisible` is `true` so it seems that is actually correctly displayed in master and not in `master_04_03_2024`. I also set `tableHeaderVisible` to `false` in JVx.mobile and then they are correctly hidden.

**Comment by gimmixAT** [comment]
cell styles should be fixed in `master` by df541ccb1b1618bc2b73760717587ff30b0f6eea

**Comment by rjahn** [comment]
header: ok - problem in master_04_03
cell profile: fixed

header in master_04_03: doesn't work correctly - but we keep focus on master
